about/balena-wpe.md

```diff
@@ -0,0 +1,136 @@
+
+1. Select the appropriate type of device, and either the Production or
+   Development flavour of the image, and download it
+
+1. Write the image to a micro-SD card or similar method appropriate
+   for your use-case
+
+
+> _Note_: In the Getting Started guide it recommends to use
+> 	  `balena-cli` tools, as it's easier to do the initial
+> 	  configuration and write the image, but other methods can be
+> 	  used.
+
+
+### Run Balena-WPE _block_ (container service)
+
+This can be done following the [Running your first Container](https://www.balena.io/os/docs/raspberrypi3/getting-started/#Running-your-first-Container)
+section in the Getting Started guide, but using the URL for the source
+of the `balena-wpe` block instead.
+
+Using `balena-cli` tools, the basic steps are:
+
+```shell
+$ git clone https://github.com/Igalia/balena-wpe.git
+
+$ cd balena-wpe
+
+$ balena push devicename.local
+```
+
+The first time it will be slow because the base images need to be
+pulled into the device, but subsequent operations will be faster.
+
+During all of these operations, the log messages are shown in the
+console where these commands were issued, to see the progress or
+possible problems.
+
+See [Service running and URLs](#service-running-and-urls) for testing.
+
+
+## Service running and URLs
+
+After the service starts, it should show a website in the screen
+connected to your device.  By default it loads
+[https://wpewebkit.org](https://wpewebkit.org), but this can be
+changed with the environment variable `WPE_URL` in the container.
+
+Additionally, this _block_ ships with
+[tohora](https://github.com/mozz100/tohora/), which is a simple
+interface to change dynamically the URL being shown.  It can be
+accessed in port 8080, something like: `http://devicename.local:8080/`.
+
+
+## Demos
+
+| Name | Animation |
+|:----|:---:|
+| CSS transformation and animations in [WebKit poster circle example](https://webkit.org/blog-files/3d-transforms/poster-circle.html) | ![Image: WebKit poster circle demo](https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/postercircle.gif) |
+| Playing an embedded video from [wpewebkit.org](https://wpewebkit.org/) | ![Image: Embedded video demo](https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/wpewebkitorg.gif) |
+| Playing [BouncyBalls](https://bouncyballs.org/) | ![Image: Playing BouncyBalls demo](https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/bouncyballs.gif) |
+| Rendering WebGL [Aquarium](https://webglsamples.org/aquarium/aquarium.html) demo | ![Image: Rendering WebGL Aquarium demo](https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/webgl_aquarium.gif) |
+| Rendering WebGL [Abstract Shapes](https://mrdoob.neocities.org/023/) | ![Image: Rendering WebGL Abstract Shapes demo](https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/webgl_abstractshapes.gif) |
```

Comment:
This URL seems to be wrong, the server returns 404 not found:

- https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/webgl_abstractshapes.gif

Reply:
That would be fixed with https://github.com/Igalia/balena-wpe/pull/16
@psaavedra , can we go ahead and merge that change?

Reply:
Wouldn't it be better to host copies of the images at `wpewebkit.org`?

Reply:
I don't have a strong opinion, so I'll include images then.

Reply:
The images can not be seen with the given syntax in Markdown in the preview site.  Should I do it with relative paths?  I think that it's better to keep it in Markdown though, and not do it with html `<img>` tag as in `about/architecture.md`

Reply:
The images show just fine here in the preview. I suppose the GitHub pages service took a little while to properly update the content 😉